README.md

```diff
@@ -401,7 +401,7 @@ following list includes some of our recent presentations:
 Are you interested in contributing to
 [GatorGrader](https://github.com/GatorEducator/gatorgrader),
 [GatorGradle](https://github.com/GatorEducator/gatorgradle), or
-[GatorGrader-SampleLab](https://github.com/GatorEducator/gatorgrader-samplelab)?
+[GatorGrader-SampleLab](https://github.com/GatorEducator/java-assigment-starter)?
```

Comment:
It would be good to note that this fix is a duplicate of #90, in fixing #74.

Reply:
Additionally, changing the displayed text here to something like `Java Sample Lab` would probably be better.

Reply:
Yeah was not sure what you all wanted to do with that.

Reply:
It might be best to remove this change for this PR, if possible.